airbyte-integrations/connector-templates/source-configuration-based/pyproject.toml.hbs

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-{{dashCase name}}"
+description = "Source implementation for {{dashCase name}}."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/{{dashCase name}}"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [ { include = "source_{{snakeCase name}}", from="src"}, {include = "main.py", from = "src"} ]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
```

Comment:
Why under 3.12?

Reply:
I think this is consistent with the other packages we have, just curious.

Reply:
`airbyte-lib` had a compatibility issue with 3.12. 
This is not set in stone though, this could be changed be developers if >=3.12 becomes a requirement